browser/ui/views/speedreader/speedreader_mode_bubble.cc

```diff
@@ -46,6 +46,8 @@ constexpr int kBubbleWidth = 324;  // width is 324 pixels
 
 constexpr int kFontSizeSiteTitle = 14;  // site title font size
 
+constexpr char kSpeedreaderSettings[] = "brave://settings/?search=speedreader";
```

Comment:
Not sure this can work for all localiazations?
If we don't translate `speedreader`, it would be fine.